src/nbl/video/alloc/GPUMemoryAllocatorBase.cpp

```diff
@@ -12,5 +12,6 @@ using namespace video;
 
 void            GPUMemoryAllocatorBase::copyBuffersWrapper(IGPUBuffer* oldBuffer, IGPUBuffer* newBuffer, size_t oldOffset, size_t newOffset, size_t copyRangeLen)
 {
-    mDriver->copyBuffer(oldBuffer,newBuffer,oldOffset,newOffset,copyRangeLen);
+    // TODO
+    //mDriver->copyBuffer(oldBuffer,newBuffer,oldOffset,newOffset,copyRangeLen);
 }
```

Comment:
this one is going to be tricky, do we give a queue to use by the allocator or do we pass explicitly all the time